ui/src/app/dashboard/grouped-summary/grouped-summary.component.css

```diff
@@ -1,14 +1,60 @@
-.mat-card {
+#container {
+  display: inline-block;
+  width: 25rem;
   margin: 1rem;
 }
 
-table a {
+div.expanded mat-card {
```

Comment:
We can ask UX about some improvements here, but I think it might be better to set a minimum card height, so that both the smaller cards and the collapsed cards have a uniform look. Thoughts?